test_helpers/utils.go

```diff
@@ -200,6 +200,14 @@ func SkipIfWatchOnceUnsupported(t *testing.T) {
 	SkipIfFeatureUnsupported(t, "watch once", 3, 0, 0)
 }
 
+// SkipIfCrudSpliceUnsupported skips test run if Tarantool without support of
+// crud splice update operation is used.
```

Comment:
crud splice is core Tarantool splice

Reply:
But for some reason, using splice in crud is not supported for Tarantool < 2.0. While we can use splice from the core.

Reply:
Can you show the exact error?

Reply:
Sure. For example:
https://github.com/tarantool/go-tarantool/actions/runs/6875782438/job/18700075402

```
    --- FAIL: TestCrudProcessData/Update (0.00s)
        tarantool_test.go:583: Failed to perform CRUD request on CRUD side: map[interface {}]interface {}{"class_name":"UpdateError", "err":"Failed to update: Unknown UPDATE operation", "file":"...ool/crud/testdata/.rocks/share/tarantool/crud/update.lua", "line":0xb0, "str":"UpdateError: Failed to update: Unknown UPDATE operation"}
```

But for the core:
```
PASS
ok  	github.com/tarantool/go-tarantool/v2	4.867s
```

Reply:
It seems that this one is more "broken" than "unsupported": https://github.com/tarantool/crud/issues/397

Thank for the report. I think it's worth to add a link to the issue to skip helper.

Reply:
Updated function name and added a link to an issue.